test/e2e/node/mount_propagation.go

```diff
@@ -101,7 +101,7 @@ var _ = SIGDescribe("Mount propagation", func() {
 		// Fail the test if the namespace is not set. We expect that the
 		// namespace is unique and we might delete user data if it's not.
 		if len(f.Namespace.Name) == 0 {
-			framework.ExpectNotEqual(f.Namespace.Name, "")
+			gomega.Expect(f.Namespace.Name).To(gomega.BeEmpty())
```

Comment:
Should it be an opposite, i.e.
```
gomega.Expect(f.Namespace.Name).NotTo(gomega.BeEmpty())
```

Reply:
P.S. and if so, why all CI tests passed with your change?

Reply:
Yes,you are right,I will check it again

Reply:
> P.S. and if so, why all CI tests passed with your change?

Here the if statement ensures that a unique namespace exists in the test environment. Under normal circumstances, the code will not enter this conditional branch，so CI did not report an error.
Thank you for pointing out the issue in my code @bart0sh